packages/@uppy/dashboard/src/components/Dashboard.js

```diff
@@ -68,6 +73,10 @@ module.exports = function Dashboard (props) {
           }
 
         <div class="uppy-Dashboard-innerWrap">
+          <div class="uppy-Dashboard-drop-files-here-hint">
+            {props.i18n('dropHint')}
```

Comment:
Tiny style nitpick: I’d call it `uppy-Dashboard-dropFilesHereHint`.

Reply:
@arturi, oh right, thanks.